cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/validation/CassandraValidation.java

```diff
@@ -0,0 +1,24 @@
+package org.apache.cassandra.spark.validation;
+
+import org.apache.cassandra.sidecar.client.SidecarClient;
+import org.apache.cassandra.sidecar.common.data.HealthResponse;
+
+public class CassandraValidation implements StartupValidation
+{
+    private final SidecarClient sidecar;
+
+    public CassandraValidation(SidecarClient sidecar)
+    {
+        this.sidecar = sidecar;
+    }
+
+    @Override
+    public void validate()
+    {
+        HealthResponse health = sidecar.cassandraHealth().get();
```

Comment:
wouldn't Cassandra Health endpoint cover both sidecar health and Cassandra's health? Do we need to perform both calls?

Reply:
We do have two distinct endpoints in Sidecar, so I think we should call both.
It's up to Sidecar to decide what "being healthy" means for it.
Can be that Cassandra is healthy, but sidecar isn't, for some reason.